Modules/_sqlite/module.c

```diff
@@ -360,6 +344,11 @@ PyMODINIT_FUNC PyInit__sqlite3(void)
     PyObject *tmp_obj;
     int i;
 
+    if (sqlite3_libversion_number() < 3007003) {
+        PyErr_SetString(PyExc_ImportError, MODULE_NAME ": SQLite 3.7.3 or higher required");
```

Comment:
Is possible to reach this point? If the shared object that loads does not have some of the functions that we are using it will be a link error, no?

Reply:
Yes, I'm pretty sure you're right about that, so this check seems superfluous.

Reply:
However, if the reason for requiring a specific version was, for example, a new flag, we'd need the runtime check. That's not the case now, so no need to keep the check. (Though it may be easier to remember to update the runtime check if it's there already.)

Reply:
Let me know if you want it removed.

Reply:
Is fine, we can leave it in case something goes terribly wrong or if some distribution packages stubs for the symbols (not sure why anybody would do that but I have seen it in the past).